stix_shifter_modules/paloalto/configuration/config.json

```diff
@@ -0,0 +1,41 @@
+{
+    "connection": {
+        "type": {
+            "displayName": "Palo Alto Cortex XDR"
+        },
+        "host": {
+            "type": "text"
+        },
+        "help": {
+            "type": "link",
+            "default": "data-sources.html"
+        },
+        "options": {
+            "type": "fields",
+            "result_limit": {
+                "default": 1000,
+                "min": 1,
+                "max": 1000,
+                "type": "number"
+            }
+        },
+        "quota_threshold": {
+            "default": 3.00,
```

Comment:
why is the default value 3.00?

Reply:
To reserve only 60% of the license quota (5) to the API call, it is set to 3. Now the user is allowed to provide values between 1 and 4 in the CP4S UI. As discussed, additional quota threshold configuration is also available.

Reply:
Can you leave a comment on this line stating that the regular license API quota is 5 so anyone seeing this knows what it's for?

Reply:
Added details in lang_en.json

Reply:
As discussed in the status call, additional quota configuration is merged with 'quota_threshold' configuration.

Reply:
As discussed, updated x-oca-asset and x-oca-event mappings.